test/compflags/unknownLlvmFlag.prediff

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+# normalize 'unknown argument:' to 'unsupported option'
```

Comment:
is that because different versions of LLVM vary between these two? If so, maybe include that in the comment?